test/built-ins/ShadowRealm/WrappedFunction/name.js

```diff
@@ -63,18 +63,20 @@ verifyProperty(wrapped, "name", {
 });
 
 // The value of fn.name is not a string.
-wrapped = r.evaluate(`
+for (const name of [null, undefined, 0, '1n', false, NaN, Infinity, 'Symbol()', '[]', '{}']) {
```

Comment:
Just a minor suggestion. It's non blocking because the `{}` already covers this well.

```suggestion
for (const name of [null, undefined, 0, '1n', false, NaN, Infinity, 'Symbol()', '[]', '{}', 'String("primitives only")']) {
```

Reply:
Do you mean `new String`?

Reply:
Thanks! I fixed my suggestion above!